lib/jack_tokenizer.rb

```diff
@@ -5,12 +5,34 @@ def initialize(io)
   end
 
   def has_more_tokens?
+    while WHITESPACES.include?(@input[@index])
+      @index += 1
+    end
+
+    if @input[@index] == "/" && @input[@index+1] == "/"
+      while @input[@index] != "\n" && @index < @input.length
+        @index += 1
+      end
```

Comment:
It’s kind of nice that this matches the shape of the other pieces of code which scan through the string, but since you’re only looking for a single character, you could also use `String#index` if you wanted to.

Reply:
👍 Done: [`9c753f3` (#5)](https://github.com/wildmaples/jack-compiler/pull/5/commits/9c753f3fa01a0c4e81bcd43c5a7735cb3e5009ac)